template/Goerli-DssSpell.sol

```diff
@@ -14,16 +14,12 @@
 // You should have received a copy of the GNU Affero General Public License
```

Comment:
also the `Goerli-DssSpellCollateral.sol` could be updated in the naming and kept for future need

Reply:
I just removed this file.